modin/experimental/core/storage_formats/pandas/native_query_compiler.py

```diff
@@ -0,0 +1,1283 @@
+# Licensed to Modin Development Team under one or more contributor license agreements.
```

Comment:
Let's move this file out of experimental folder to the core.

Reply:
Done.